### PR TITLE
Add Real file path and allow to refresh image gallery

### PR DESCRIPTION
**Pull request by CircleCurve**
Fixed some image picker would not refresh gallery when capture or save the new image. And also, identifier only get the path which is content://....etc . And one more option to allow real path

### CODE REVIEW

File: android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java [hunk 1]
```diff
@@ -222,15 +235,72 @@ private void openImagePicker() {
...
+                context.sendBroadcast(new Intent(Intent.ACTION_MEDIA_MOUNTED, Uri.parse("file://" + Environment.getExternalStorageDirectory())));
+                finishWithSuccess(true);
+            }
+        } catch (Exception e) {
+            finishWithError("unknown error", e.toString());
+        }
+    }
+
+    /*
+    private void refreshGallery(final MultiImagePickerPlugin.Refresh refresh) {
```
Comment (Sh1d0w) [inline android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java]: Can you please remove this if not needed.

File: android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java [hunk 2]
```diff
@@ -222,15 +235,72 @@ private void openImagePicker() {
...
+        Matisse.from(MultiImagePickerPlugin.this.activity)
                 .choose(MimeType.ofImage())
                 .countable(true)
                 .maxSelectable(maxImages)
                 .restrictOrientation(ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED)
                 .imageEngine(new GlideEngine())
                 .forResult(REQUEST_CODE_CHOOSE);
+
+        /*
+        this.refreshGallery(new Refresh() {
```
Comment (Sh1d0w) [inline android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java]: Can you please remove this if not needed.

File: android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java [hunk 3]
```diff
@@ -10,12 +10,14 @@
...
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
+import android.media.MediaScannerConnection;
 import android.media.ThumbnailUtils;
 import android.net.Uri;
 
 import com.zhihu.matisse.Matisse;
 import com.zhihu.matisse.MimeType;
 import com.zhihu.matisse.engine.impl.GlideEngine;
+import com.zhihu.matisse.internal.entity.CaptureStrategy;
```
Comment (Sh1d0w) [inline android/src/main/java/com/vitanov/multiimagepicker/MultiImagePickerPlugin.java]: Do we need this import?

File: android/.idea/.name [hunk 4]
```diff
@@ -0,0 +1 @@
+multi_image_picker
```
Comment (Sh1d0w) [inline android/.idea/.name]: Can you please add the whole .idea folder to .gitignore and remove it from this commit. We don't need it to be in the repository.

**Comment by CircleCurve** [comment]
I have remove an unrelated comment code and add .idea in .gitignore file. Thanks for your kindly help

**Comment by Sh1d0w** [comment]
The PR has been merged and I've pushed new release 2.1.25, which now can be installed.

Thanks!

**Pull request merged**